eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -64,6 +68,10 @@ object TimestampQueryFilters {
   }
 }
 
+object SignedMessage {
+  def signedBytes(message: ByteVector): ByteVector32 = Crypto.hash256(ByteVector("Lightning Signed Message:".getBytes) ++ message)
```

Comment:
lnd answered me on Twitter and told me that Go guarantees that the conversion from string to bytes will always use UTF8 regardless of the machine's locale, so we should force UTF8 for the prefix:

```suggestion
  def signedBytes(message: ByteVector): ByteVector32 = Crypto.hash256(ByteVector("Lightning Signed Message:".getBytes(StandardCharsets.UTF_8)) ++ message)
```

Reply:
Done